src/chrome/content/rules/Maybank.co.id.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+
+
+	Non-functional subdomains:
+
+		- mail	(m)
+		- maingw2	(t)
+		- www.mbiwfh	(i)
+		- microsite	(m)
```

Comment:
Can be redirected to `www.micorsite`
See also `form.maybank.co.id` (requires exclusion and custom test url)

Reply:
Added redirect. All the pages I found on `form.maybank.co.id` (via google) simply timeout.